internal/backend/auth/authloginhttpsvc/svc.go

```diff
@@ -24,28 +27,27 @@ type Deps struct {
 	fx.In
 
 	Muxes    *muxes.Muxes
-	Z        *zap.Logger
+	L        *zap.Logger
 	Cfg      *Config
 	DB       db.DB
 	Sessions authsessions.Store
 	Tokens   authtokens.Tokens
+	Users    authusers.Users
 }
 
-type svc struct {
-	Deps
-
-	loginMatcher func(string) bool
-}
+type svc struct{ Deps }
 
 func Init(deps Deps) error {
-	svc := &svc{
-		Deps:         deps,
-		loginMatcher: compileLoginMatchers(strings.Split(deps.Cfg.AllowedLogins, ",")),
-	}
-
+	svc := &svc{Deps: deps}
 	return svc.registerRoutes(deps.Muxes)
 }
 
+type loginData struct {
+	Email        string
+	ProviderName string
+	ProviderData any
```

Comment:
maybe map of string:string is better ?

Reply:
we don't really do anything with this rn, so i'd just leave it as is, or maybe remove it?

Reply:
if you can remove it even better

Reply:
removed